rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtils.java

```diff
@@ -91,10 +97,29 @@ protected void prePopulateRDTPatientFields(Patient patient, JSONObject field) th
         } else if (PATIENT_AGE.equals(key)) {
             field.put(JsonFormUtils.VALUE, patient.getAge());
         }
+
+        // pre-populate the patient detail name
+        if (CovidConstants.FormFields.PATIENT_INFO_NAME.equals(field.getString(KEY))) {
+            fillPatientData(field, patient.getPatientName());
+        }
+
+        // pre-populate the patient detail dob
+        if (CovidConstants.FormFields.PATIENT_INFO_DOB.equals(field.getString(KEY))) {
+            fillPatientData(field, patient.getDob());
+        }
```

Comment:
Shouldn't these fields be populated in the sample collection form?

Reply:
Also, could you add unit tests to verify all new fields are correctly pre-populated?

Reply:
yes, fixed.

Reply:
This is the [relevance file](https://github.com/onaio/rdt-standard/blob/master/rdt-open-reader/src/covidIndonesia/assets/rule/sample-collection-form-relevance-rules.yml) for sample collection form. 

A file can be [marked](https://github.com/onaio/rdt-standard/blob/dc90494ea4ae8ada194408b62fc13390988126d9/rdt-open-reader/src/covidIndonesia/assets/json.form/sample-collection-form-original.json#L162) as depending on relevance rules.

Reply:
Extra documentation on relevance can be found in the native forms library documentation.

Reply:
`CovidRDTJsonFormUtilTest` updated.

Reply:
The tests would need to verify (assert) that the patient details fields have the correct value pre-populated. I don't think this exists yet.